app/src/main/java/uk/ac/ebi/atlas/configuration/CacheConfig.java

```diff
@@ -42,6 +42,7 @@ public CacheManager cacheManager() {
 
                 builder -> builder.name("hcaMetadata"),
                 builder -> builder.name("inferredCellTypesOntology"),
-                builder -> builder.name("inferredCellTypesAuthors"));
+                builder -> builder.name("inferredCellTypesAuthors"),
+                builder -> builder.name("experimentAccessions"));
```

Comment:
I guess a better name for this cache would be `privateExperimentAccessions`

Reply:
yeah, Thanks. I will fix it.

Reply:
Fixed it.